lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -316,6 +331,24 @@ export default class LockMonitor {
     return true;
   }
 
+  /**
+   * Performs the release lock routine. Returns true if there were changes to the lock; false otherwise.

```

Comment:
Super nit: consider using `@returns` for describing return values. I was looking for it when reading code, didn't realized it was actually described it in summary.

Reply:
Sure will do.